src/core/lib/channel/channel_args.h

```diff
@@ -84,14 +85,14 @@ using RefType = absl::remove_cvref_t<decltype(*std::declval<T>().Ref())>;
 template <typename T>
 struct ChannelArgTypeTraits<
     T, absl::enable_if_t<
-           std::is_base_of<RefCounted<channel_args_detail::RefType<T>>,
-                           channel_args_detail::RefType<T>>::value ||
+           std::is_base_of<RefCounted<channel_args_detail::RefType<T> >,
```

Comment:
did clang-format do this?

Reply:
That may have been my IDE, I made some changes there. But clang_format_code doesn't fix it, interestingly.